packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -333,6 +334,24 @@ class ResizeHandle extends (Widget as any) {
       );
     }
 
+    eventsEngine.on(
```

Comment:
I suggest splitting the method into several smaller methods, each adding a subscription corresponding to its specific functionality